elide-model-config/src/main/resources/elideTableSchema.json

```diff
@@ -1,454 +1,426 @@
+                },
+                "definition": {
+                    "title": "Join definition SQL",
+                    "description": "Templated SQL expression that represents the ON clause of the join",
+                    "type": "string"
+                }
+            },
+            "required": [
+                "name",
+                "to",
+                "type",
+                "definition"
+            ],
+            "additionalProperties": false
         },
-        "tags": {
-          "title": "Dimension tags",
-          "description": "An array of string based tags for dimensions",
-          "type": "array",
-          "uniqueItems": true,
-          "items": {
-            "type": "string"
-          },
-          "default": []
-        }
-
-      }
-    },
-    "dimension": {
-      "title": "Dimension",
-      "description": "Dimensions represent labels for measures. Dimensions are used to filter and group measures.",
-      "type": "object",
-      "allOf": [
-        { "$ref": "#/definitions/dimensionRef" },
-        {
-          "properties": {
-            "type": { "$ref": "#/definitions/columnTypes" },
-            "values": {
-              "title": "Dimension values",
-              "description": "An array of valid string values for this dimension",
-              "type": "array",
-              "uniqueItems": true,
-              "items": {
-                "type": "string"
-              },
-              "default": []
+        "measure": {
+            "title": "Measure",
+            "description": "Metric definitions are extensible objects that contain a type field and one or more additional attributes. Each type is tied to logic in Elide that generates a metric function.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "title": "Metric name",
+                    "description": "The name of the metric. This will be the same as the POJO field name.",
+                    "type": "string",
+                    "format": "elideFieldName"
+                },
+                "description": {
+                    "title": "Metric description",
+                    "description": "A long description of the metric.",
+                    "type": "string"
+                },
+                "category": {
+                    "title": "Measure group category",
+                    "description": "Category for grouping",
+                    "type": "string"
+                },
+                "hidden": {
+                    "title": "Hide/Show measure",
+                    "description": "Whether this metric is exposed via API metadata",
+                    "type": "boolean",
+                    "default": false
+                },
+                "readAccess": {
+                    "title": "Measure read access",
+                    "description": "Read permission for the metric.",
+                    "type": "string",
```

Comment:
Do we validate these at all?

Reply:
no, it will fail at later stage if its wrong. We can probably add validation in DynamicConfigValidator.